### PR TITLE
Fix bugs from 100.8 regressions testing 

### PR DESCRIPTION
**Pull request by sclaridge**
This PR includes an enhancement for the viewshed samples and fixes some bugs from 100.8 regression testing:

- Removes the colour picker from Viewshed (location) sample and explicitly sets the colours of the visible, obstructed areas and frustum outline. This prevents the changes from the color picker from effecting the other viewshed samples in the sample viewer.
- Removes the unclickable duplicate cell from the transformations list in List transformations by suitability sample.
- Adds CSS colours to the labels in Control annotation sublayer visibility. Previously in the sample viewer, if you hovered over the open sublayer checkbox label when out of scale, it would switch from gray to white. 

Thanks!

@JonLavi

### CODE REVIEW

File: analysis/viewshed-location/src/main/java/com/esri/samples/viewshed_location/ViewshedLocationController.java
Comment (JonLavi) [inline analysis/viewshed-location/src/main/java/com/esri/samples/viewshed_location/ViewshedLocationController.java]: Good solution to the issue, to just go ahead and simplify it 👍 
We only need to update the screenshot to use the colours that we're setting here.

File: display_information/control-annotation-sublayer-visibility/src/main/resources/control_annotation_sublayer_visibility/style.css
Comment (JonLavi) [inline display_information/control-annotation-sublayer-visibility/src/main/resources/control_annotation_sublayer_visibility/style.css]: This only really applies to `closedSublayerCheckbox` right now, so I think we could specify by writing:
```suggestion
#closedSublayerCheckbox {
```
Of course this depends on whether this works ok with the SV, so best test it out!

File: geometry/list-transformations-by-suitability/src/main/java/com/esri/samples/list_transformations_by_suitability/ListTransformationsBySuitabilitySample.java
Comment (JonLavi) [inline geometry/list-transformations-by-suitability/src/main/java/com/esri/samples/list_transformations_by_suitability/ListTransformationsBySuitabilitySample.java]: Good spot on this, it was quite a sneaky bug!

**Review by JonLavi — APPROVED** [review]
Thanks for fixing these @sclaridge .
Just a couple of suggestions. The only thing really needs updating is the screenshot for `ViewshedLocation`. 
Approving so that you can move on with the second review 👍

File: display_information/control-annotation-sublayer-visibility/src/main/resources/control_annotation_sublayer_visibility/style.css
Comment (sclaridge) [reply display_information/control-annotation-sublayer-visibility/src/main/resources/control_annotation_sublayer_visibility/style.css]: This works in the SV!

**Comment by sclaridge** [comment]
@JonLavi Thanks for the feedback, I've update the screenshot! 

Based on feedback from the samples team, I've added a small change so the frustum outline is visible when the sample starts up.

File: display_information/control-annotation-sublayer-visibility/src/main/java/com/esri/samples/control_annotation_sublayer_visibility/ControlAnnotationSublayerVisibilitySample.java
Comment (Rachael-E) [inline display_information/control-annotation-sublayer-visibility/src/main/java/com/esri/samples/control_annotation_sublayer_visibility/ControlAnnotationSublayerVisibilitySample.java]: since this is the colour being set to the checkbox as the sample loads, this should be able to go into the .css file in the same way that the closedSublayerCheckbox is, e.g. 

```
#openSublayerCheckbox  {
   -fx-text-fill: darkgrey;
}
```

File: geometry/list-transformations-by-suitability/src/main/java/com/esri/samples/list_transformations_by_suitability/ListTransformationsBySuitabilitySample.java
Comment (Rachael-E) [inline geometry/list-transformations-by-suitability/src/main/java/com/esri/samples/list_transformations_by_suitability/ListTransformationsBySuitabilitySample.java]: naming conventions for booleans can be vague with Java, but we can try and be more succinct with their naming here. something like `isEmpty` would be more readable here.
```suggestion
        protected void updateItem(DatumTransformation transformation, boolean isEmpty) {
```

**Review by Rachael-E — COMMENTED** [review]
hey @sclaridge , thanks for pulling this together! I have two minor suggestions, and I'll come back to the Viewshed colour part after we hear back from the review tomorrow after our chat. Thanks!

File: geometry/list-transformations-by-suitability/src/main/java/com/esri/samples/list_transformations_by_suitability/ListTransformationsBySuitabilitySample.java
Comment (sclaridge) [reply geometry/list-transformations-by-suitability/src/main/java/com/esri/samples/list_transformations_by_suitability/ListTransformationsBySuitabilitySample.java]: @Rachael-E At our review meeting on Friday, the samples team confirmed that we'll remove the colour picking functionality. Setting the colours explicitly in this sample shows off the ability to configure the colours of the viewshed areas without impacting the other samples.

**Review by Rachael-E — APPROVED** [review]
thanks @sclaridge for taking that to review, this is good to go ahead and merge then!

**Pull request merged**
